cargo-dist/src/config.rs

```diff
@@ -435,6 +435,11 @@ pub struct DistMetadata {
     /// Any additional steps that need to be performed before building local artifacts
     #[serde(default)]
     pub github_build_setup: Option<String>,
+
+    /// Configuration specific to Mac .pkg installers
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
+    pub mac_pkg_config: Option<MacPkgConfig>,
```

Comment:
glossing over this as something we can freely rejig in config 1.0...

Reply:
Yep, like I mentioned elsewhere this is something I wanted to check in with you about wrt handling before/after config 1.0.